src/node.rs

```diff
@@ -333,6 +354,67 @@ impl<S: StorageExt> SawtoothRaftNode<S> {
             }
         }
     }
+
+    fn check_for_conf_change(&mut self, block_id: BlockId) -> Option<ConfChange> {
+        // Get list of peers from settings
+        let settings = self.service
+            .get_settings(block_id, vec![String::from("sawtooth.consensus.raft.peers")])
+            .expect("Failed to get settings");
```

Comment:
The error message that was deleted `"'sawtooth.consensus.raft.peers' must be set to use Raft"` seemed more informative.

Reply:
That error message will be displayed if the peers setting isn't found (in the call to `get_peers_from_settings`). This is a different error where it can't even get the settings; if this fails, it's due to an actual error reported by the service.

Reply:
What happens when the validator goes down?  Does Raft go down too?

Reply:
Yes, Raft goes down as well.

Reply:
It sounds like this is currently a common pattern in sawtooth-raft, when dealing with network communication with the validator.  We should create a story to clean this up and handle any places where an error could occur from the validator gracefully.